docs/source/conf.py

```diff
@@ -320,7 +320,7 @@ def inject_weight_metadata(app, what, name, obj, options, lines):
       used within the autoclass directive.
     """
 
-    if obj.__name__.endswith(("_Weights", "_QuantizedWeights")):
+    if getattr(obj, ".__name__", "").endswith(("_Weights", "_QuantizedWeights")):
```

Comment:
you can ignore this. It's because `obj` was sometimes a `property` method coming from the Datapoint classes, and these `property` objects don't have `.__name__`.